### PR TITLE
executor,planner: support show create view

### PR DESCRIPTION
**Pull request by AndrewDi**
### What problem does this PR solve? 
As part of view feature, add `SHOW CREATE VIEW` grammar support.

### What is changed and how it works?
ref proposal [Proposal: Implement View](https://github.com/pingcap/tidb/blob/master/docs/design/2018-10-24-view-support.md)

### Check List 

Tests 
 - Unit test

### CODE REVIEW

**Comment by AndrewDi** [comment]
@XuHuaiyu @zz-jason @crazycs520 PTAL

File: executor/errors.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ var (
 	ErrDBaccessDenied              = terror.ClassExecutor.New(mysql.ErrDBaccessDenied, mysql.MySQLErrName[mysql.ErrDBaccessDenied])
 	ErrTableaccessDenied           = terror.ClassExecutor.New(mysql.ErrTableaccessDenied, mysql.MySQLErrName[mysql.ErrTableaccessDenied])
 	ErrBadDB                       = terror.ClassExecutor.New(mysql.ErrBadDB, mysql.MySQLErrName[mysql.ErrBadDB])
+	ErrTableIsNotView              = terror.ClassExecutor.New(mysql.ErrWrongObject, "'%s.%s' is not VIEW")
```
Comment (zz-jason) [inline executor/errors.go]: should we use the error messages from `mysql.MySQLErrName`?

Comment (AndrewDi) [reply executor/errors.go]: This error code may be a little different, `mysql.ErrWrongObject` has different message, witch is `xxx is not view` and `xxx is not base table`, and `mysql.MySQLErrName` didn't contain these messages.

Comment (zz-jason) [reply executor/errors.go]: In MySQL:
```sql
MySQL(root@localhost:test) > show create view t;
ERROR 1347 (HY000): 'test.t' is not VIEW
```

Notice that the error code is `1347`, after searching in MySQL code base, the error pattern is:
```
{ "ER_WRONG_OBJECT", 1347, "\'%-.192s.%-.192s\' is not %s","HY000", "", 316 },
```

So in https://github.com/pingcap/tidb/pull/9309/files#diff-df919d9361195b93310b219738ea1aeeR764, I think the error we should use is:
```go
ErrWrongObject.GenWithStackByArgs(db.Name.O, tb.Meta().Name.O, "VIEW")
```

Comment (AndrewDi) [reply executor/errors.go]: You are right, I will fix all this kind error handle.

**Comment by AndrewDi** [comment]
@zz-jason Does common-test has broken?

**Comment by zz-jason** [comment]
> @zz-jason Does common-test has broken?

oops, let me take a look

**Comment by zz-jason** [comment]
@AndrewDi The CI is broken by https://github.com/pingcap/tidb/issues/9331. We'll try to fix it as soon as possible.

**Pull request merged**